aspnetcore/grpc/health-checks.md

```diff
@@ -60,7 +60,7 @@ The service name specified by the client is usually the default (`""`) or a pack
 
 ### Configure health checks execution interval
 
-Health checks are executed immediately when `Check` is called. The `Watch` is a streaming method has different behavior. The long running stream reports health checks results over time by periodically executing <xref:Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheckPublisher> to gather health results. By default, the publisher waits 5 seconds after app startup before running health checks, and then health checks are run again every 30 seconds.
+Health checks are executed immediately when `Check` is called. `Watch` is a streaming method and has a different behavior than `Check`: The long running stream reports health checks results over time by periodically executing <xref:Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheckPublisher> to gather health results. By default, the publisher waits 5 seconds after app startup before running health checks, and then health checks are run again every 30 seconds.
```

Comment:
Style guide: avoid execute, use only when needed.

Long sentences are difficult to MT and difficult to parse by native readers.

```suggestion
Health checks are run immediately when `Check` is called. `Watch` is a streaming method and has a different behavior than `Check`: The long running stream reports health checks results over time by periodically executing <xref:Microsoft.Extensions.Diagnostics.HealthChecks.IHealthCheckPublisher> to gather health results. By default, the publisher:

* Waits 5 seconds after app startup before running health checks.
* Runs health checks every 30 seconds.
```

Reply:
That change is under `:::moniker range=">= aspnetcore-6.0"`

Reply:
No need. This applies to all versions.